user-guide/Standard_Apps/IDP/Installing_DataMiner_IDP.md

```diff
@@ -10,9 +10,11 @@ To install and set up DataMiner IDP:
 
     - DataMiner version 10.0.0 CU9 or higher must be installed in your DMS.
 
-    - If you want to use scheduled activities, DataMiner Indexing must be installed in your DMS.
+    - If you want to use scheduled activities, your DMS must use an Elasticsearch database.
 
-    - If you want to use Process Automation, DataMiner SRM, Process Automation and Token activity must be installed. If you want to be able to repeat processes with a timer, you also need to install the Repeat Gateway.
+    - If you want to use Process Automation, then DataMiner SRM, Process Automation, and Token activity must be installed. If you want to be able to repeat processes with a timer, you also need to install the Repeat Gateway.
+
+    - The SRM package must be installed before IDP is installed.
```

Comment:
In fact, it's not only the SRM package that needs to be installed before IDP. It's SRM, PA, Token Activity and (when applicable) the Repeat Gateway.

Reply:
Thanks! I've updated the text based on your feedback.